docs/source/potentiostat_operation.rst

```diff
@@ -0,0 +1,55 @@
+.. This page is designed to guide users through steps to setup the software and run experiments.
+
+.. _potentiostat-operation:
+
+How to Operate the JUAMI Potentiostat
+=====================================
+
+Once you are familiar with the electrochemical characterization techniques available through the JUAMI potentiostat
+and how to setup an electrochemical experiment, follow the guide below that provides instructions on how to setup and run
+the JUAMI potentiostat. These instructions assume that the sample or device of interest is ready to be connected and
+tested (all that needs to be done is clip the leads to the electrodes). See :ref:`technique-background` and
+:ref:`experiment-setup` for more information.
+
+Start the Pytentiostat Experiment
+---------------------------------
+
+First, ensure that the JUAMI potentiostat is connected to a USB port via a USB-B to USB-A adapter. To begin an experiment,
+open anaconda prompt from the start menu. Navigate to the pytentiostat/ directory and run main.py.
```

Comment:
Like Simon said on other PR should be more general than anaconda prompt

Reply:
This is something I could use everyone's help with. I am not well-versed in command prompt syntax. I can learn Windows, but someone will need to give me details for Mac. Right now I am able to run the code through PyCharm and Anaconda Prompt.

@aplymill7 @sbillinge @jlhitt1993